File: TestWCF/TestWCF2/ICalculator.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.Serialization;
using System.ServiceModel;
using System.Text;

namespace TestWCF2
{
    // 注意: 如果更改此处的接口名称 "ICalculator"，也必须更新 Web.config 中对 "ICalculator" 的引用。
    [ServiceContract]
    public interface ICalculator
    {
        [OperationContract]
        double Add(double n1, double n2);
        [OperationContract]
        double Subtract(double n1, double n2);
        [OperationContract]
        double Multiply(double n1, double n2);
        [OperationContract]
        double Divide(double n1, double n2);
    }
}

File: TestWCF/TestWCF/ICalculator.vb
Imports System.ServiceModel

' 注意: 如果更改此处的类名 "ICalculator"，也必须更新 Web.config 中对 "ICalculator" 的引用。
<ServiceContract()> _
Public Interface ICalculator

    <OperationContract()> _
    Sub DoWork()

End Interface
